### DISCUSSION TITLE
Rename .tex files

### DISCUSSION THREAD

**Discussion by kuschuermann** [texstudio-org/texstudio#3051] [Ideas]
It would be nice if I could rename files (perhaps with a right-click popup menu option) shown in the structure viewer, and this would also update the content of the related root document.

The reason: I work on a document with a fairly large number of \input{...} elements. Sometimes I change my mind about a referenced filename and wish I could just rename it on the spot. Instead, I must first open a filesystem viewer, rename the file, and then fix the broken references in TeXstudio.

**Comment by sunderme** [comment]
file/Fifix ?

**Reply by kuschuermann** [reply]
Ah, that, too wasn't available in my older version. I'll give that a try. Thank  you.
